addons/crm/models/crm_lead.py

```diff
@@ -1231,13 +1231,16 @@ def get_empty_list_help(self, help):
             help_title = _('Create a new lead')
         else:
             help_title = _('Create an opportunity to start playing with your pipeline.')
-        alias_record = self.env['mail.alias'].search([
-            ('alias_name', '!=', False),
-            ('alias_name', '!=', ''),
-            ('alias_model_id.model', '=', 'crm.lead'),
-            ('alias_parent_model_id.model', '=', 'crm.team'),
-            ('alias_force_thread_id', '=', False)
-        ], limit=1)
+        alias_domain = [
+            ('company_id.id', '=', self.env.company.id),
+            ('alias_id.alias_name', '!=', False),
+            ('alias_id.alias_name', '!=', ''),
```

Comment:
Could at least filter on creation model isn't it ? Even if team will filter afterwards, we may have quite a lot of aliases e.g. with accounting journals creating their own.

Reply:
Doesn't hurt 

I'd specifically suggested to remove it because we're searching on crm.team now, so it should join on `crm_team.alias_id = mail_alias.id` and the results would normally all create leads implicitly. But I guess it's technically possible for the alias_id to create some other model for some reason so yeah.

Reply:
At some point they could create sale orders for example, there was some "configuration" steps on sales team (lead, opportunity, sales, ...). It disappeared but yep, you could use sales team to create another kind of document automatically.